patterns/pattern-2/src/classification_function/index.py

```diff
@@ -34,6 +35,12 @@ def handler(event, context):
     # Use default=str to handle Decimal and other non-serializable types
     logger.info(f"Config: {json.dumps(config, default=str)}")
     
+    # Extract document from the OCR result - handle both compressed and uncompressed
```

Comment:
Adding duplicated code here..    Please carefully check all the changes for errors like this.

<img width="954" height="222" alt="image" src="https://github.com/user-attachments/assets/094afc31-c6ca-4a3b-ba70-f94a0fae0519" />